crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -571,11 +571,11 @@ where
                         // FCU resulted in a fatal error from which we can't recover
                         let err = err.clone();
                         let _ = tx.send(Err(error));
-                        return OnForkchoiceUpdateOutcome::Fatal(err)
+                        return OnForkchoiceUpdateOutcome::Fatal(err);
                     }
                 }
                 let _ = tx.send(Err(error));
-                return OnForkchoiceUpdateOutcome::Processed
+                return OnForkchoiceUpdateOutcome::Processed;
```

Comment:
```suggestion
                return OnForkchoiceUpdateOutcome::Processed
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/145